batch/batch/cloud/gcp/resource_utils.py

```diff
@@ -39,6 +39,160 @@ def __init__(self, machine_family: str, worker_type: str, cores: int, memory: in
 
 
 MACHINE_TYPE_TO_PARTS = {
+    'n1-standard-1-gpu': MachineTypeParts(
```

Comment:
since you can have different gpus, I recommend using a pattern like
```
{machine)-{configuration}-{mem}-{cpus}-{accelerator}-{naccel}
```
example: n1-standard-4-t4-1